web/src/scripts/webflow/fundingSync.mjs

```diff
@@ -296,15 +308,21 @@ const createNewFundings = async () => {
     try {
       return await createItem(funding, fundingCollectionId, false);
     } catch (error) {
+      if (error.response.status === 429){
```

Comment:
[pebble] This should have been caught by automatic code formatting: there should be a space between `)` and `{`. Are you formatting the code before commit?

Reply:
It should do that automatically right? Our commit script should run prettier I believe. Maybe mine isn't working for some reason? Or maybe it doesn't run on the .mjs files?